rllib/models/temp_spec_classes.py

```diff
@@ -4,48 +4,12 @@
 This file is to be removed once these modules are commited to master.
 """
 
-from typing import Any
+from ray.rllib.utils.nested_dict import NestedDict
```

Comment:
Let's remove this temp folder entirely, right? TensorDict is `NestedDict[Tensor]`. Just use that.

Reply:
You can define `TensorDict = NestedDict[Tensor]` in rllib.typing btw.

Reply:
We still need the `ModelConfig` for now, I think.